README.md

```diff
@@ -6,7 +6,7 @@ This gem combines the speed of `libsass`, the [Sass C implementation](https://gi
 
 ### libsass Version
 
-[3.6.1](https://github.com/sass/libsass/releases/3.6.1)
+[3.6.4](https://github.com/sass/libsass/releases/3.6.4)
```

Comment:
re: https://github.com/sass/sassc-ruby/commit/38cbb789037bf58301e512ff8bd59f43f3c89d6a – addresses cognitive dissonance in README